### PR TITLE
Update pull_request_template.md

### PR DESCRIPTION
**Pull request by buxx**
Proposition of new merge request template update.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -2,12 +2,20 @@
...
+**For code reviewer**
+
 - [ ] Code is clear enough
 - [ ] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
 - [ ] If there are TODO, NOTE or HACK in code, date and developer initials are present
 - [ ] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation
 
+**For developper**
+
+- [ ] Manual tests done to ensure stability on whole application layers, issue expectation follow
```
Comment (buxx) [inline .github/pull_request_template.md]: What do you think about add one:

    - [ ] Original issue have been updated with a short resume of implemented solution

Comment (inkhey) [reply .github/pull_request_template.md]: great idea.

**Comment by PhilippeAccorsi** [comment]
@buxx I think you can add "Original issue ......".
After that we can merge to make better MR.

File: .github/pull_request_template.md (hunk 1 above)
Comment (PhilippeAccorsi) [reply .github/pull_request_template.md]: @buxx can you add this ?

**Pull request merged**